utils/UniswapPoolsHelpers.ts

```diff
@@ -113,5 +114,13 @@ export async function getPools(hre: HardhatRuntimeEnvironment): Promise<{ name:
     }
   }
 
+  const bdus = await getBdUs(hre);
+
+  const supplementaryUsdStableAddress = constants.SUPPLEMENTARY_USD_STABLE_ADDRESS(hre.network.name);
+  const supplementaryUsdStableName = constants.SUPPLEMENTARY_USD_STABLE_NAME(hre.network.name);
+  const usdc = (await hre.ethers.getContractAt("IERC20", supplementaryUsdStableAddress)) as IERC20;
```

Comment:
we will sure `registerPool()` also for RSK and other networks, so we need this variable (usdc) to be generic